windows-gaming-driver/src/driver/controller.rs

```diff
@@ -37,10 +38,11 @@ enum State {
     Resuming,
 }
 
-#[derive(PartialEq, Eq, Clone, Copy)]
+#[derive(Clone)]
```

Comment:
This not being `Copy` breaks every single `match`, doesn't it?

Even if it's not as nice, storing the temporary-entry-sender in an `Option` field in `Controller` is probably much better.